pkg/server/v3/server_test.go

```diff
@@ -578,6 +839,62 @@ func TestAggregatedHandlers(t *testing.T) {
 	}
 }
 
+func TestDeltaAggregatedHandlers(t *testing.T) {
+	config := makeMockConfigWatcher()
+	config.deltaResponses = makeDeltaResponses()
+	resp := makeMockDeltaStream(t)
+
+	resp.recv <- &discovery.DeltaDiscoveryRequest{
+		Node:    node,
+		TypeUrl: rsrc.ListenerType,
+	}
+	// Delta compress node
```

Comment:
Unclear what this means from a skim, remove/elaborate

Reply:
Removed 👍